frontend/tests/playwright/product/add_to_cart.spec.ts

```diff
@@ -3,15 +3,14 @@ import { mockedProductQuery } from '@tests/jest/__mocks__/products';
 import { cloneDeep } from 'lodash';
 
 test.describe('product page add to cart', () => {
-   test.beforeEach(async ({ page }) => {
-      await page.route(
-         '**/api/2.0/internal/international_store_promotion/buybox*',
-         (route) => {
-            route.fulfill({
-               status: 200,
-               body: '',
-            });
-         }
+   test.beforeEach(async ({ clientRequestHandler, rest }) => {
+      clientRequestHandler.use(
+         rest.post(
+            '/api/2.0/internal/international_store_promotion/buybox',
+            (req, res, ctx) => {
+               return res(ctx.status(200), ctx.json({}));
```

Comment:
One question, why do we do `ctx.json({})` here but for the `notifyWhenSkuInStock` request below we don't add the body?

Reply:
Good question! 

I have no clue 😅 , I think I missed the return body when migrating it over.

deploy_block ☁️

Reply:
Good question! 

I have no clue 😅 , I think I missed the return body when migrating it over.

deploy_block ☁️

Reply:
So the `ctx.json({})` was actually a bug, and we should not have returned a body. The reason this continued to work was that it was trying to match against a `post` instead of a `get` and therefore the original request was still sent. 

I didn't think this was an issue until addressing #1278 where I discovered that this request was the reason the international buybox was rendering if my IP was located in Europe.

As such, I made additional changes to properly match against this request and fix #1278